colcon_sanitizer_reports/_sanitizer_section_part.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from collections import ChainMap, defaultdict
+import re
+from typing import List, Mapping, Pattern, Tuple
+
+from colcon_sanitizer_reports._sanitizer_section_part_stack_trace import \
+    SanitizerSectionPartStackTrace
+
+
+_FIND_RELEVANT_STACK_TRACE_BEGIN_REGEXES_BY_ERROR_NAME: Mapping[str, Tuple[Pattern[str], ...]] = (
+    ChainMap({
+        # There are two relevant stack traces involved in a "data race" section part. Their headers
+        # match the following patterns.
+        'data race': (
+            re.compile(r'^\s+(Read|Write) of size \d+ at 0x[\da-f]+ .*$'),
```

Comment:
Just to double check: where in the code does this package make sure that the correct locale is being used in order to successfully match these language specific output? If it doesn't do so I recommend to catch this case and disable the extension with a warning message.

Reply:
Added an issue for this: https://github.com/colcon/colcon-sanitizer-reports/issues/11